stock-tracker/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@types/socket.io-client": "^1.4.32",
```

Comment:
maybe better to change it to devDependencies

Reply:
good spot, will change that now!